packages/astro/src/core/config/config.ts

```diff
@@ -351,3 +357,9 @@ export function mergeConfig(
 ): Record<string, any> {
 	return mergeConfigRecursively(defaults, overrides, isRoot ? '' : '.');
 }
+
+// TODO: remove after the experimetal phase when
+// it won't be needed to edit two config fields to enable the feature
+function isHybridMalconfigured(config: AstroConfig) {
+	return config.experimental.hybridOutput ? config.output !== 'hybrid' : config.output === 'hybrid';
```

Comment:
This is an example of another function that might be nice to live in a prerender folder.